doc/changelog/08-tools/10577-extraction-dependent-projections.rst

```diff
@@ -0,0 +1,8 @@
+- Fix a printing bug of OCaml extraction on dependent record projections, which
+  produced improper `assert false`. This change makes the OCaml extractor
+  internally inlines record projections by default and may not produce record
```

Comment:
I think the correct grammar would be:

```suggestion
  internally inline record projections by default and may not produce record
```

Reply:
@Zimmi48 Sorry, I didn't reply on this but actually I fixed this too. Could you confirm it?

Reply:
@pi8027 Sure, I didn't really intend to go back to this PR and check. I believe the assignee should be @maximedenes here anyway.